js/client-chat.js

```diff
@@ -1099,6 +1099,16 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'uhtml':
+					this.$chat.append('<div class="notice uhtml-' + row[1] + '">' + Tools.sanitizeHTML(row.slice(2).join('|')) + '</div>');
```

Comment:
Also did you forget the part where I said `toId(row[1])`? For this and for `uhtmlchange` too.